### PR TITLE
fix: setup.py locations

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

File: scripts/tests.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ test_dir=$(pwd)/tests
 
 pip install wheel
 pip install -r tests/requirements.txt
+export PYTHONPATH=.
```
Comment (davidbp) [inline scripts/tests.sh:11]: why did you add this? Is it needed? I don't recall other executors to export the python path

Comment (cristianmtr) [reply scripts/tests.sh:11]: tests would not find `jinahub` otherwise

Comment (JoanFM) [reply scripts/tests.sh:11]: I think is much safer to add `try` `catch` to import from local and then from jinahub

Comment (cristianmtr) [reply scripts/tests.sh:11]: It would fail anyway.

Comment (cristianmtr) [reply scripts/tests.sh:11]: Tried removing it and the test failed.

**Review by davidbp — APPROVED** [review]
LGTM
